packages/proxy/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@packages/proxy",
+  "version": "0.0.0",
+  "private": true,
+  "main": "index.js",
+  "scripts": {
+    "build-js": "bin-up tsc --project .",
+    "clean-deps": "rm -rf node_modules",
+    "test": "bin-up mocha --reporter mocha-multi-reporters --reporter-options configFile=../../mocha-reporter-config.json"
+  },
+  "dependencies": {
+    "bluebird": "3.5.3",
+    "charset": "^1.0.1",
```

Comment:
Should set dependencies to exact installed version, not `^` as per our dependency rules.